src/goto-programs/validate_goto_model.cpp

```diff
@@ -169,28 +169,40 @@ void validate_goto_modelt::check_called_functions()
   class test_for_function_addresst : public const_expr_visitort
   {
   public:
-    std::set<irep_idt> identifiers;
-
+    test_for_function_addresst(
+      const function_mapt &function_map,
+      const validation_modet &vm)
+      : function_map{function_map}, vm{vm}
+    {
+    }
     void operator()(const exprt &expr) override
```

Comment:
Blank lines between function definitions are free of charge ;-)

Reply:
:) Done.